src/Sarif.Driver.UnitTests/ExceptionRaisingRule.cs

```diff
@@ -105,12 +105,19 @@ public void Analyze(TestAnalysisContext context)
 
             if (_exceptionCondition == ExceptionCondition.ParsingTarget)
             {
-                Errors.LogTargetParseError(context, null, "Could not parse target.");
+                Errors.LogTargetParseError(
+                    context,
+                    new Region
+                    {
+                        StartLine = 42,
+                        StartColumn = 54
+                    },
+                    "Could not parse target.");
```

Comment:
Should LogTargetParse error insist on a non-null region? Probably not.. bug to consider fixing?

---
In reply to: [61528567](https://github.com/Microsoft/sarif-sdk/pull/229#discussion_r61528567) [](ancestors = 61528567)